ChangeLog

```diff
@@ -9,6 +9,8 @@ Release date: TBA
 ..
   Put new features here
 
+* Fix issue with ``--min-similarity-lines`` used with ``--jobs``.
```

Comment:
I think this should be moved up in the latest version :) Could you also rebase on the latest master, please ?

Reply:
Done.